hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/WriteOperationHelper.java

```diff
@@ -538,7 +538,7 @@ public CompleteMultipartUploadResult commitUpload(
   public BulkOperationState initiateCommitOperation(
       Path path) throws IOException {
     return S3Guard.initiateBulkWrite(owner.getMetadataStore(),
-        BulkOperationState.OperationType.Put, path);
```

Comment:
I hadn't defined the commit type, I'd just had a "Put". Now I've added that and an Fsck. It's useful when you log the s3guard.Operations log (Try it!) because we log the bulk operation Id and type as well as DDB PUT/DELETE/TOMBSTONE operations. Worth doing if you want to see what's being done to a table

Reply:
Ok understood,  I saw later that this is new.